Packer/ArkCase-EE-Core/setup-metering.service

```diff
@@ -3,7 +3,7 @@ Description=Setup metering reporting
 
 [Service]
 Type=oneshot
-ExecStart=/usr/local/bin/setup-metering.sh
+ExecStart=/bin/bash /usr/local/bin/setup-metering.sh
```

Comment:
Not necessary

Reply:
I'm agree with you @fabricetriboix . Tnx